### PR TITLE
#5 :memo: some initial documentation tweaks

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**

### CODE REVIEW

File: foundation/foundation-maven-plugins/artifacts-maven-plugin/pom.xml [hunk 1]
```diff
@@ -1,40 +1,41 @@
 <?xml version="1.0"?>
```
Comment (d-ryan-ashcraft) [inline foundation/foundation-maven-plugins/artifacts-maven-plugin/pom.xml:1]: I (Informational): fixes build resolution where cucumber is pulling a `-SNAPSHOT` version of the `io.cucumber:messages:19.1.4` and breaking the build (may work if you are pointing to a Maven mirror that pulls in SNAPSHOTs subtly - but will not work on strict public repos)

**Comment by nartieri** [comment]
Content and updates look good. Approving to merge.

File: docs/README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
...
+    ```bash
+    npx gulp bundle
+   ```
+2. The UI bundle will be available at `build/ui-bundle.zip`.
+   _Note: this zip will need to be committed after the changes have been made_
+
+For further information on packaging for previewing, check out Antora [docs](https://docs.antora.org/antora-ui-default/build-preview-ui/).
+
+## Releasing a Fixed Version
+TODO: update with content
```
Comment (meliz19) [inline docs/README.md:56]: Is this TODO meant to be here?

Comment (d-ryan-ashcraft) [reply docs/README.md:56]: Yes - will be layered in in further commits on this ticket.  But good question - appreciate you checking on it!

**Pull request merged**